### PR TITLE
Use signed packages and remove internal feed

### PR DESCRIPTION
**Pull request by genlu**
FYI @dibarbet @sharwell

### CODE REVIEW

**Review by JoeRobich — APPROVED** [review]

**Review by dibarbet — APPROVED** [review]

File: src/CodeStyle/CSharp/Tests/Microsoft.CodeAnalysis.CSharp.CodeStyle.UnitTests.csproj [hunk 1]
```diff
@@ -56,5 +56,9 @@
     <ProjectReference Include="..\..\..\Compilers\Test\Core\Microsoft.CodeAnalysis.Test.Utilities.csproj" />
     <ProjectReference Include="..\..\..\Workspaces\CoreTestUtilities\Microsoft.CodeAnalysis.Workspaces.Test.Utilities.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <!-- Add explicit reference to avoid NU1603 missing package warning -->
+    <PackageReference Include="Microsoft.VisualStudio.Interop" Version="$(MicrosoftVisualStudioInteropVersion)"/>
```
Comment (sharwell) [inline src/CodeStyle/CSharp/Tests/Microsoft.CodeAnalysis.CSharp.CodeStyle.UnitTests.csproj:61]: ❔ Is there a bug tracking this?

Comment (genlu) [reply src/CodeStyle/CSharp/Tests/Microsoft.CodeAnalysis.CSharp.CodeStyle.UnitTests.csproj:61]: No, I'm hoping to follow up on this once interop branch is merged into regular dev17 branch (hopefully in a few days)

**Comment by genlu** [comment]
@JoeRobich I think we can reenable CI gates for this branch, now that the internal feed is removed

**Comment by JoeRobich** [comment]
> @JoeRobich I think we can reenable CI gates for this branch, now that the internal feed is removed

@genlu enabled

**Pull request merged**